collector/prometheus_collector.go

```diff
@@ -184,6 +184,14 @@ func (s byName) Less(i, j int) bool {
 	return s.prometheusLabels[i].GetName() < s.prometheusLabels[j].GetName()
 }
 
+func prometheusLabelSetToCadvisorLabels(promLabels model.Metric) map[string]string {
```

Comment:
is this actually necessary?  It seems like as long as the underlying type of model.Metric is a map[string]string, we can assign it to map[string]string.

I tested this out: https://play.golang.org/p/htMzF1q0_df

Reply:
```
   255				// TODO Handle multiple labels nicer. Prometheus metrics can have multiple
   256				// labels, cadvisor only accepts a single string for the metric label.
   257				label := prometheusLabelSetToCadvisorLabel(sample.Metric)
   258	                        var labels map[string]string
   259				labels = sample.Metric
   260	
   261				metric := v1.MetricVal{
   262					FloatValue: float64(sample.Value),
   263					Timestamp:  sample.Timestamp.Time(),
   264					Label:      label,
   265					Labels:     labels,
   266				}

```
```
/go/src/github.com/google/cadvisor # make build
>> building assets
>> building binaries
>> building cadvisor
# github.com/google/cadvisor/collector
collector/prometheus_collector.go:259:11: cannot use sample.Metric (type model.Metric) as type map[string]string in assignment
make: *** [Makefile:43: build] Error 2
/go/src/github.com/google/cadvisor # go version
go version go1.13.5 linux/amd64
```

seems necessary, it's explicit type casting. I'm not expert in go, may be exist better solution

Reply:
https://play.golang.org/p/7AJ9BIbiS-M

Reply:
Huh, neat.  That is fine then.